lib/hammer_cli_foreman/realm.rb

```diff
@@ -0,0 +1,53 @@
+module HammerCLIForeman
+  class Realm < HammerCLIForeman::Command
+
+    resource :realms
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+      end
+
+      build_options
+    end
+
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output ListCommand.output_definition do
+        field :realm_proxy_id, _("Realm proxy id")
```

Comment:
Just one nitpick: I haven't tested the API, but if it returns `realm_proxy_name` too, it would be good to use
```ruby
field nil, _("Realm Proxy"), Fields::SingleReference, :key => :realm_proxy
```
instead, to show name by default and id when user asks for it (`--show-ids`).